<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>266</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono ExtraLight</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Nombre semi-premier</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_verif">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>90</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>JetBrains Mono ExtraLight</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>VÃ©rifier</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_res">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>431</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>435</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>JetBrains Mono Light</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Introduire un entier &gt; 2: </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="le_nbre">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
